fl4health/servers/adaptive_constraint_servers/mrmtl_server.py

```diff
@@ -12,6 +13,7 @@ class MrMtlServer(FlServer):
     def __init__(
         self,
         client_manager: ClientManager,
+        fl_config: Config,
```

Comment:
I may be missing something but parent (FlServer) takes in a few other parameters like model? Is there a reason we are omitting those arguments in the subclasses like MrMtl and others